clients/go-ethereum:stable/Dockerfile

```diff
@@ -10,8 +10,9 @@ RUN apk add --update bash jq
 
 # Build go-ethereum on the fly and delete all build tools afterwards
 RUN \
-  apk add --update go git make gcc musl-dev                                        && \
-  git clone --depth 1 --branch release/1.5 https://github.com/ethereum/go-ethereum && \
+  apk add --update bash jq go git make gcc musl-dev             \ 
+  	  ca-certificates linux-headers                           && \
```

Comment:
Please ensure the `master` and `stable` Dockerfiles of go-ethereum are similar. You patched in the same fixes in different ways into them.